client/src/app/queries/facts.ts

```diff
@@ -0,0 +1,34 @@
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+
+import { getFacts } from "@app/api/rest";
+import { AxiosError } from "axios";
+import { Application, Fact } from "@app/api/models";
+import { ApplicationsQueryKey } from "./applications";
+
+export const FactsQueryKey = "facts";
+
+export const useFetchFacts = (applicationID: number | string | undefined) => {
+  const { data, isLoading, error, refetch } = useQuery(
+    [FactsQueryKey, applicationID],
+    {
+      queryFn: async () => await getFacts(applicationID),
```

Comment:
I noticed this recently in a lot of our queries (and I am guilty of copying it), but we don't actually need async/await for these queryFns if all we're doing is calling a function that already returns a promise. We may want to simplify all these at some point to:
```ts
queryFn: () => getFacts(applicationID)
```
however this is harmless as-is, just a nitpick.